jest.config.js

```diff
@@ -2,5 +2,10 @@ module.exports = {
   preset: 'ts-jest',
   testRegex: '/test/.*\\.test\\.tsx$',
   modulePathIgnorePatterns: ['<rootDir>/examples/'],
-  setupFilesAfterEnv: ['<rootDir>/jest-setup.ts']
+  setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
+  globals: {
+    'ts-jest': {
+      diagnostics: false
```

Comment:
`diagnostics` is removed in #903, I think we want to keep it removed

Reply:
find a batter way to disable strict mode in test 👌